drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/projects_importer.rb

```diff
@@ -27,7 +38,11 @@ def run!
       timeout_seconds = 60
       success = false
       Hmis::HmisBase.with_advisory_lock(JOB_LOCK_NAME, timeout_seconds: timeout_seconds) do
-        _run
+        # transform data files to HUD 2024
+        Dir.mktmpdir do |hud_dir|
+          CsvTransformer2022to2024.up(dir, hud_dir)
+          _run(hud_dir)
+        end
```

Comment:
@ttoomey it slipped my mind that this file _is_ going to change into 2024 format at some point, and we need to be prepared to handle that. Normally there is an Export.csv which specifies which version it is, but that's not provided in this case, so we can just check the project columns ( code below). If the current format is 2024, we should skip the transformation.
Sorry I didn't mention that earlier!



```
    def determine_hud_version
      headers = records_from_csv('Project.csv', row_limit: 1).first.to_h.keys
      if headers.include?('RRHSubType')
        '2024'
      else
        '2022'
      end
    end
```

Reply:
sounds like a good idea, I added some conditions to detect 2022 and raise/log to sentry